fdbclient/FDBTypes.h

```diff
@@ -740,16 +740,18 @@ struct TLogVersion {
 		// V4 changed how data gets written to satellite TLogs so that we can peek from them;
 		// V5 merged reference and value spilling
 		// V6 added span context to list of serialized mutations sent from proxy to tlogs
+		// V7 use xxhash3 for TLog checksum
 		// V1 = 1,  // 4.6 is dispatched to via 6.0
 		V2 = 2, // 6.0
 		V3 = 3, // 6.1
 		V4 = 4, // 6.2
 		V5 = 5, // 6.3
 		V6 = 6, // 7.0
+		V7 = 7, // 7.2
```

Comment:
I think this should be 7.1

Reply:
Discussed with @sfc-gh-satherton offline and we think it may not worth the effort to patch 7.0 for this change. So we will make the new format readable in 7.1 and default in 7.2.